libs/sdk-core/src/swap_in/swap.rs

```diff
@@ -381,14 +383,16 @@ impl BTCReceiveSwap {
                 .update_swap_paid_amount(bitcoin_address.clone(), payment.unwrap().amount_msat)?;
         }
 
-        Ok(self.persister.update_swap_chain_info(
-            bitcoin_address,
-            utxos.unconfirmed_sats(),
-            utxos.unconfirmed_tx_ids(),
-            utxos.confirmed_sats(),
-            utxos.confirmed_tx_ids(),
-            swap_status,
-        )?)
+        let chain_info = SwapChainInfo {
+            unconfirmed_sats: utxos.unconfirmed_sats(),
+            unconfirmed_tx_ids: utxos.unconfirmed_tx_ids(),
+            confirmed_sats: utxos.confirmed_sats(),
+            confirmed_tx_ids: utxos.confirmed_tx_ids(),
+            confirmed_at: confirmed_block,
```

Comment:
I see a bug here where the confirmed_at is not optional and may be zero.

Reply:
Good catch, I thought at this point the known